### ISSUE TITLE
Map all OTel attributes

### ISSUE THREAD

**Issue by noamalffasy** [elastic/apm-data#183]
As of now, only some of the attributes are mapped.

Some attributes (like `process.command_args`, `process.owner`, `network.*`) don't have any mapping and are automatically stored under `labels`.

Are there any plans to map them all? Or are there only specific ones that are supposed to be mapped?

**Comment by axw** [comment]
Yes. We intend to change how we store OTel data, and in the future attributes should be stored as-is. e.g. `process.owner` would be stored as `process.owner`, and would not be rewritten as `labels.process_owner`.

**Comment by josh-williams** [comment]
@axw Is there any update on this? I'm trying to set up all Elastic collection via OTel/APM server for an application I'm working on, and I can't really do that if I can't use most Semantic Convention attributes, so I might have to switch to another collection method if this isn't coming soon.

**Comment by axw** [comment]
@josh-williams for ingesting OTel data without loss of fidelity, we're now more focused on contributions to OpenTelemetry Collector, and the [elasticsearch exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/elasticsearchexporter). We may still make this change to APM Server, but it's not likely to come as soon as originally anticipated.

Just last week, support for indexing OTLP logs into Elasticsearch in a new "OTel-native" document structure was merged: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33290. Attribute names are used as-is, embedded within a namespace such as `resource.attributes` or `scope.attributes`.

Support for indexing metrics with this document structure is WIP (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34248), traces should be a fast follow, and then profiles.
